src/main/java/com/puppycrawl/tools/checkstyle/checks/header/AbstractHeaderCheck.java

```diff
@@ -77,19 +77,21 @@ protected List<String> getHeaderLines() {
     }
 
     /**
-     * Setter to specify the charset to use when reading the headerFile.
+     * Setter to specify the character encoding to use when reading the headerFile.
      *
      * @param charset the charset name to use for loading the header from a file
+     * @since 4.4
      */
     public void setCharset(String charset) {
         this.charset = createCharset(charset);
     }
 
     /**
-     * Setter to specify the name of the file containing the required header..
+     * Setter to specify the name of the file containing the required header.
      *
      * @param uri the uri of the header to load.
      * @throws CheckstyleException if fileName is empty.
+     * @since 4.4
```

Comment:
Same.

Reply:
all since are reverted. map is updated with values from xdoc